webapp/components/EmotionButton/EmotionButton.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="emotion-button">
+    <base-button :id="emotion" circle ghost @click="$emit('toggleEmotion', emotion)">
+      <img :src="emojiPath" />
```

Comment:
the pics are not showing up...
![Screenshot from 2020-01-15 15-21-54](https://user-images.githubusercontent.com/26943915/72442543-015d8f00-37ad-11ea-8b13-107293131295.png)

I played around with it, and adding a max width to the `img`, they look good to me 
![Screenshot from 2020-01-15 15-43-02](https://user-images.githubusercontent.com/26943915/72442971-c871ea00-37ad-11ea-86b0-825b8891184b.png)
![Screenshot from 2020-01-15 15-43-30](https://user-images.githubusercontent.com/26943915/72442982-cc057100-37ad-11ea-9d98-90d08d9044fe.png)

I'll push up the code... you can change, if you have a better solution...

Reply:
maybe I should start testing things in Chrome... 😝